main.nf

```diff
@@ -253,7 +252,6 @@ if (params.sentieon) log.warn "[nf-core/sarek] Sentieon will be used, only works
 ================================================================================
 */
 
-
 include { BWAMEM2_MEM }           from './modules/local/bwamem2_mem.nf'
 include { GET_SOFTWARE_VERSIONS } from './modules/local/get_software_versions'
```

Comment:
```suggestion
include { GET_SOFTWARE_VERSIONS } from './modules/local/process/get_software_versions'
```

Reply:
I have changed the directory structure to be able to add in `local/subworkflows` too. See [local modules](https://github.com/drpatelh/chipseq/tree/master/modules/local) and [nf-core/modules](https://github.com/drpatelh/chipseq/tree/master/modules/nf-core).